packages/item/test/item-mixin.test.js

```diff
@@ -125,37 +151,44 @@ describe('vaadin-item-mixin', () => {
       });
 
       it('should have focus-ring attribute when focused with keyboard', () => {
-        fire(item, 'focus');
+        tabKeyDown(item);
+        focusin(item);
         expect(item.hasAttribute('focus-ring')).to.be.true;
       });
 
-      it('should not have focus-ring after blur', () => {
-        fire(item, 'focus');
-        fire(item, 'blur');
+      it('should not have focus-ring after focusout', () => {
+        tabKeyDown(item);
+        focusin(item);
+        focusout(item);
         expect(item.hasAttribute('focus-ring')).to.be.false;
       });
-    });
 
-    describe('interaction', () => {
-      it('set this._mousedown to false if mouseup was outside', () => {
-        fire(item, 'mousedown');
-        expect(item._mousedown).to.be.true;
-        fire(document, 'mouseup');
-        expect(item._mousedown).to.be.false;
+      it('should set focus-ring on programmatic focus after keydown', () => {
+        tabKeyDown(item);
+        item.focus();
+        expect(item.hasAttribute('focus-ring')).to.be.true;
       });
 
-      it('should fire click event when activated with Enter', () => {
+      it('should not set focus-ring on programmatic focus after mousedown', () => {
+        mousedown(item);
```

Comment:
I would suggest that the `tabKeyDown(item)` method is called before `mousedown` to make the test not depend on other tests.

Reply:
Thanks, done.